acceptance-tests/src/test/java/linea/plugin/acc/test/PluginTest.java

```diff
@@ -266,6 +294,50 @@ public void deniedTransactionNotAddedToTxPool() throws Exception {
         .isEqualTo("destination address is a precompile address and cannot receive transactions");
   }
 
+  @Test
+  public void transactionIsNotMinedWhenTooManyTraceLines() throws Exception {
```

Comment:
this test name doesn't seem to match the code

Reply:
The test sends 5 transactions, each of these transactions keeps the trace lines for the add module under the limit, but a second of these transactions pushes the block over the limit.
The test makes sure that all transactions are mined, but only one per block.

Reply:
I think that's a helpful comment to add...

Reply:
```suggestion
  public void secondTransactionIsNotIncludedInBlock_WhenTooManyTraceLines() throws Exception {
```

Reply:
or maybe `onlyOneTransactionFitsInABlock`